tensorflow/core/util/tensor_bundle/byte_swap_tensor.cc

```diff
@@ -161,6 +161,14 @@ absl::Status ByteSwapTensor(Tensor* t) {
                         t->NumElements());
 }
 
+absl::Status ByteSwapTensorProto(TensorProto* tp) {
+  char* buff = const_cast<char*>(std::string(tp->tensor_content()).data());
```

Comment:
Can you actually save the string to a variable, please? We're getting this internal error:

```
tensorflow/core/util/tensor_bundle/byte_swap_tensor.cc:165:34: error: object backing the pointer will be destroyed at the end of the full-expression [-Werror,-Wdangling-gsl]
```

Looks like a temporary lifetime issue

Reply:
Done. It simplifies setting the content back into the proto too.

Thanks for your help with the internal errors @mihaimaruseac!